python/lsst/summit/utils/auxtel/mount.py

```diff
@@ -0,0 +1,74 @@
+# This file is part of summit_utils.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import logging
+
+import numpy as np
+
+from ..efdUtils import getEfdData
+
+
+def hasTimebaseErrors(expRecord, client, maxDiff=1.05):
+    """Check if an exposure has cRIO timebase errors.
+
+    Data in the lsst.sal.ATMCS.mount_AzEl_Encoders topic is a packed
+    time-series, and if the amount of data that is packed in isn't packed for a
+    period of almost exactly one second then the unpacking code will
+    incorrectly assign the intra-second timestamps, causing misalignment, as
+    this breaks the fundamental assumption required to have a packed
+    timeseries.
+
+    Parameters
+    ----------
+    expRecord : `lsst.daf.butler.dimensions.DimensionRecord`
+        The exposure record to query.
+    client : `lsst_efd_client.efd_helper.EfdClient`
+        The EFD client to use.
+    maxDiff : `float`, optional
+        The maximum difference in cRIO timestamps to consider as a timebase
```

Comment:
Units are seconds? Also, is this the absolute value of the difference?

Reply:
Yup, will add comment on being seconds, though it's hard-coded in the cRIO, so it's somewhat irrelevant and would need re-crafting for other topics depending on how they're packed.